api/storage_service.cc

```diff
@@ -1390,25 +1390,23 @@ void set_snapshot(http_context& ctx, routes& r, sharded<db::snapshot_ctl>& snap_
             });
         }
 
-        sstables::compaction_type_options::scrub opts = {
-            .operation_mode = scrub_mode,
-        };
         const sstring quarantine_mode_str = req_param<sstring>(*req, "quarantine_mode", "INCLUDE");
+        sstables::compaction_type_options::scrub::quarantine_mode quarantine_mode;
         if (quarantine_mode_str == "INCLUDE") {
```

Comment:
This patch seems unrelated.

Reply:
It prepares the ground for passing passing lower-level arguments to scrub compaction. Currently the options struct mixes high level and low level options, which I have to separate before doing that.